### ISSUE TITLE
[Chart.Js] - date adapter not being loaded

### ISSUE THREAD

**Issue by pcmmf** [symfony/ux#754]
Hi,
I'm using:
- Symfony 6.2.7
- symfony/ux-chartjs 2.7
- Chart.JS 3.4.1
- chartjs-adapter-luxon 1.3.1
- npm 9.5.1 (via Docker image `FROM node:19-alpine3.16`)

... with Webpack Encore for building. My `app.ts` is as follows:

```typescript
// Import CSS
import './css/app.css';

// Required for ChartJS time series axis
import { Chart } from 'chart.js';
import 'chartjs-adapter-luxon';

// Start the Stimulus application
import './bootstrap';
```

When I try to render a Chart.js chart (using a Symfony UX component), I receive a JavaScript console error of:

```
stimulus.js:2017 Error connecting controller

Error: This method is not implemented: Check that a complete date adapter is provided.
    at abstract (chart.esm.js:2488:1)
    at DateAdapter.formats (chart.esm.js:2495:1)
    at TimeScale.init (chart.esm.js:10343:12)
    at chart.esm.js:5551:1
    at each (helpers.segment.js:105:1)
    at Chart.buildOrUpdateScales (chart.esm.js:5529:9)
    at Chart._updateScales (chart.esm.js:5676:1)
    at Chart.update (chart.esm.js:5637:1)
    at new Chart (chart.esm.js:5426:1)
    at extended.connect (controller.js:18:23)
```

I've tried to run `composer update` and `npm update` to no avail.

I've previously used the Luxon adapter when I wasn't using the Symfony UX component, so I'm fairly confident it should do what I need. However, something isn't getting glued together when the adapters are loaded for Chart.Js.

Has anyone else encountered this issue?

**Comment by pcmmf** [comment]
Still picking over the bones of this, but it's possibly related to the order in which we run `composer install` and `npm install`. 

Running `npm install` first with an empty vendor folder creates `vendor/symfony/ux-chartjs/assets/node_modules/chart.js/dist/chart.esm.js` for Chart.js version 3.8.2.

Running `composer install` removes the `vendor/symfony/ux-chartjs/assets/node_modules/` folder. The charts display as expected.

If I reverse the order and do `composer install` first then `npm install`, I get the error with the date adapter not loading.

If I manually force to use chart.js v3.8.2 (`npm install chart.js@3.8.2`) everything works.

I can't yet tell where the conflicting versions are gettng set, or Chart.js is getting downgraded to 3.4.1. However, that seems to be the key.

**Comment by weaverryan** [comment]
I bet this is fixed by #870

**Comment by pcmmf** [comment]
Ah nice, thank you! I'll keep an eye out for the fix.
